src/strategies/moxie/index.ts

```diff
@@ -0,0 +1,190 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, subgraphRequest } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'Hrishikesh-Thakkar';
+export const version = '0.0.1';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const MOXIE_VESTING_SUBGRAPH_URL = "https://api.studio.thegraph.com/query/88457/moxie-vesting/version/latest";
+const MOXIE_PROTOCOL_SUBGRAPH_URL = "https://api.studio.thegraph.com/query/88457/moxie-protocol/version/latest";
+const MOXIE_LIQUIDITY_POOL_SUBGRAPH_URL = "https://api.studio.thegraph.com/query/88457/moxie-liquidity/version/latest";
```

Comment:
Hi @Hrishikesh-Thakkar These URLs are not meant for production use, could you publish and share the URLs of the subgraphs? i can update the code with our proxy server